etc/config.json

```diff
@@ -15,7 +15,8 @@
     ],
     "backend": {
         "elastic": "http://4.3.2.1:9200/",
```

Comment:
I will remove this one in a following commit in the same branch. Right now there is code, that still uses this